database/migrations/2019_10_23_142210_create_faults_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use App\FaultsTable;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateFaultsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('faults', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->unsignedBigInteger('user_id');
```

Comment:
probably should have a different name like `reporter_id`, eh or something even better.

Reply:
I agree, changed that.